modules/fluid_properties/include/utils/FPNumbers.h

```diff
@@ -0,0 +1,61 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef FPNUMBERS_H
+#define FPNUMBERS_H
+
+#include "libmesh/libmesh_common.h"
+using namespace libMesh;
+
+/**
+ * Compute Reynolds number
+ *
+ * @param volume_fraction  The volume fraction of the phase
+ * @param rho  The density of the phase
+ * @param vel  The velocity of the phase
+ * @param D_h  The hydraulic diameter
+ * @param visc  The viscosity of the phase
+ * @return Reynolds number
+ */
+Real Reynolds(Real volume_fraction, Real rho, Real vel, Real D_h, Real visc);
+
+/**
+ * Compute Prandtl number
+ *
+ * @param cp  Specific heat
+ * @param mu  Dynamic viscosity
+ * @param k  Thermal conductivity
+ * @return Prandtl number
+ */
+Real Prandtl(Real cp, Real mu, Real k);
+
+/**
+ * Compute Grashof number
+ *
+ * @param beta  Thermal expansion coefficient
+ * @param dT  |T_w - T|
```

Comment:
It might be safer to have two different arguments for these temperatures. Also, I'm not sure that `T_w` is a good name - Usually I see `dT = T_s - T_bulk`, where `T_s` is surface temperature.

Reply:
Yeah, I wanted to avoid more changes in the existing codes.